src/main/webapp/species.css

```diff
@@ -0,0 +1,54 @@
+#species-page {
+    display: flex;
+    flex-direction: row;
+    height: 100vh;
+    padding: 0 !important;
+}
+
+#species-image-container {
+    background: #AEDFDB;
+    flex-grow: 1;
+}
+
+#species-name, #species-image, #canvas, #pixel_factor {
+    display: block;
+    margin: auto;
+}
+
+#species-info-container {
+    background: #75C9CC;
+    flex-grow: 3;
+}
+
+canvas {
+    image-rendering: optimizeSpeed;
+    image-rendering: -moz-crisp-edges;
+    image-rendering: -webkit-optimize-contrast;
+    image-rendering: -o-crisp-edges;
+    image-rendering: crisp-edges;
+    -ms-interpolation-mode: nearest-neighbor;
+}
+
+li {
+  float: left;
+}
+
+li a {
+  display: block;
+  color: white !important;
+  padding: 16px;
+  text-align: center;
+  text-decoration: none;
+}
+
+li a:hover {
+  background-color: #111111;
```

Comment:
This is mostly a note for future development, but we should change all the colors to CSS variables so we can manage the color scheme easily later.